.organizationmap

```diff
@@ -718,7 +720,6 @@ UnitedStack <contact@unitedstack.com> Kun Huang <academicgareth@gmail.com>
 UnitedStack <contact@unitedstack.com> Li Tianqing <tianqing@unitedstack.com>
 UnitedStack <contact@unitedstack.com> Ning Yao <yaoning@unitedstack.com>
 UnitedStack <contact@unitedstack.com> wangsongbo <wangsongbo@unitedstack.com>
-UnitedStack <contact@unitedstack.com> Weibing Zhang <zhangweibing@unitedstack.com>
```

Comment:
see http://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Mail-notification-to-the-author-about-an-update, maybe you can split this change into another commit.

if Webbing no longer affiliated to UnitedStack, maybe you can just update `.peoplemap` ? see http://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Affiliation-change-over-time